docker-compose-tests.yml

```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+
+services:
+
+  test-etl:
+    extends:
+      file: docker-compose.yml
+      service: base-etl
+    command: yarn test
+    volumes:
+      - etl-node_modules:/code/node_modules
+    environment:
+      COVERALLS_REPO_TOKEN: ${COVERALLS_REPO_TOKEN}
+      NODE_ENV: development
+      SNYK_TOKEN: ${SNYK_TOKEN}
+      TRAVIS: ${TRAVIS}
```

Comment:
I don't believe any of the `TRAVIS` env vars are required for the `test-etl` service as it is only intended to be run locally.